### PR TITLE
Radar visualization clean up

### PR DESCRIPTION
**Pull request by nmhaddadND**
Fix the names, colors, and some ugly code. This PR fixes the naming issues associated with the different types of radars, as well as poor color choices for differentiating between sensors. It also removes global variables which can affect performance.

### CODE REVIEW

**Comment by nmhaddadND** [comment]
waiting to merge this in after approval for @jallen-NextDroid to push a branch with performance fixes

**Review by bcasselsND — APPROVED** [review]

**Pull request merged**
